inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/feature/ConceptFeatureSupport.java

```diff
@@ -120,6 +125,10 @@ public boolean accepts(AnnotationFeature aFeature)
     @Override
     public String renderFeatureValue(AnnotationFeature aFeature, String aLabel)
     {
+        FeatureValuePair pair = new FeatureValuePair(aFeature, aLabel);
+        if (renderValueCache.containsKey(pair)) {
```

Comment:
It is even faster to call "get" and then to check if the returned value is null. That way only one lookup in the cache is necessary instead of two (containsKey/get).